app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestService.kt

```diff
@@ -39,35 +39,38 @@ class MeetingRequestService : FirebaseMessagingService() {
     when (title) {
       "MEETING REQUEST" -> {
```

Comment:
Great use of when cases to make clean and well-organized code !

when (title) {
    "MEETING REQUEST" -> { /* Logic here */ }
    "MEETING RESPONSE" -> { /* Logic here */ }
    "MEETING CONFIRMATION" -> { /* Logic here */ }
    "MEETING CANCELLATION" -> { /* Logic here */ }
    "ENGAGEMENT NOTIFICATION" -> { /* Logic here */ }
}